src/System.Management.Automation/resources/RunspaceInit.resx

```diff
@@ -117,6 +117,9 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="EnabledExperimentalFeatures" xml:space="preserve">
+    <value>Variable to hold the enabled experimental feature names</value>
```

Comment:
Should we add final dot?

Reply:
I followed the existing examples. I don't have a strong opinion. It looks OK to me without the dot:
```none
PS:8> $v.Description
Parent folder of the host application of the current runspace
```